lib/chainsync/chainsync/dashboard/usernames.py

```diff
@@ -0,0 +1,116 @@
+"""Helper functions for mapping addresses to usernames."""
+import pandas as pd
+
+
+def combine_usernames(username: pd.Series) -> pd.DataFrame:
+    """Map usernames to a single user (e.g., combine click with bots)."""
+    # TODO Hard coded mapping, should be a config file somewhere
+    user_mapping = {
```

Comment:
now that we're done with the trading competition would it make sense to move this information to a config file that is not tracked by GH? So instead of hard-coding the mapping we can load a file and construct it programatically. Then in future iterations we can modify the files as needed w/o committing code.

Reply:
I'm fine with this coming in a follow-up PR